geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionClear.java

```diff
@@ -362,6 +367,10 @@ void doClear(RegionEventImpl regionEvent, boolean cacheWrite) {
       try {
         Set<Integer> bucketsCleared = clearRegion(regionEvent);
 
+        if (!partitionedRegion.allServerVersionsSupportPartitionRegionClear()) {
```

Comment:
Optional: We could have `allServerVersionsSupportPartitionRegionClear` or whatever it's named return a list of incompatible servers. Then you could format a message for PartitionedRegionVersionException which lists out the server names that that were incompatible.

Reply:
why are we doing the check here again? I though we only need to check it before we send out the message.

Reply:
Good point. We should probably pull it. Look for another checkin that will have it.

Reply:
I actually pulled the other one and kept the one in doClear.